fr/health/centreon-agent.md

```diff
@@ -0,0 +1,503 @@
+    ```
+
+### Activer la collecte de logs Centreon
+
+À partir de la version 2 de **centreon-agent**, il est possible de récupérer les logs générés par le composant Centreon supervisé. 
+
+Pour définir quels logs doivent être récupérés, vous devez créer des fichiers yml de configuration dans le dossier suivant : `/etc/centreon-agent/conf.d`.
+Pour récupérer un log précis, le fichier de configuration doit contenir les arguments suivants : `path`, `type` et `pattern` du log choisi. Exemple :
+
+```
+- path: /var/log/centreon-gorgone/gorgoned.log
+  pattern: "%{CENTREONGORGONE}"
+  type: file
+```
+
+Vous pouvez avoir plusieurs fichiers de configuration (chaque fichier est parsé et les fichiers de logs définis sont ajoutés à la collecte).
+
+#### Utiliser les modèles
+
+Pour simplifier la configuration de la collecte de logs, des modèles pré-configurés sont fournis. Chaque modèle couvre un périmètre spécifique en fonction du composant Centreon, de sa version, etc.
+
+Les modèles sont situés dans le répertoire suivant :
+
+```
+/usr/share/centreon-agent/examples
+```
+
+Suivant le composant Centreon supervisé, vous pouvez simplement copier-coller le modèle correspondant dans votre répertoire `/etc/centreon-agent/conf.d`.
+
+#### Finaliser la configuration des modèles
+
+>Pour les collecteurs Centreon, les fichiers de log sont préfixés du nom du collecteur. Vous devez donc adapter le modèle.
+>Ouvrez le modèle `poller` et remplacez tous les noms génériques `POLLERNAME` dans la section `path` par le vrai nom du collecteur.
+
+Les modèles fournis fonctionneront directement avec une installation Centreon standard. En cas de doute, vous pouvez localiser le fichier de log désiré et comparer son chemin avec celui indiqué dans la section `path` du modèle.
+
+En cas d'erreurs, vous trouverez des explications détaillées du problème dans les logs de **centreon-agent** dans `/var/log/centreon-agent/centreon-agent.log`.
+
+#### Démarrer la collecte des logs
+
+Une fois la collecte de vos logs configurée, redémarrez l'agent en utilisant la commande suivante :
+
+```
+systemctl restart centreon-agent.service
+```
+
+### Tags
+
+L'Agent peut contextualizer la collecte de données avec vos propres tags personnalisés afin de définir son périmètre d'action. Ces tags seront utilisés par la suite pour agréger les données de supervision et créer des tableaux de bord ou des rapports dans des contextes pertinents.
+
+>Nous recommandons fortement que le premier tag que vous définissiez soit “environment” afin de disposer d'une référence commune à tous les utilisateurs.
+
+Les tags peuvent être configurés dans le fichier YAML `/etc/centreon-agent/centreon-agent.yml` généré à l'installation :
+
+```yaml
+collect:
+  tags:
+    environment: [staging|preproduction|production|valeur-personnalisée]
+    [tag2]: [valeur-personnalisée2]    
+    [tag3]: [valeur-personnalisée3]
+```
+
+```yaml
+collect:
+  tags:
+    environment: production
+    City: Paris   
```

Comment:
I have a doubt wether the use of tags with uppercase letters could have an impact or not on the cloud side @Nohzoh